File: Frontend/Admin/src/app/modules/models/parteneriat.ts
export class Parteneriat {
  idParteneriat: number;
  nomParteneriat: string;
  numeroTelephone: number;
  email: string;
  description: string;
  budget:number;
  verified:boolean;

  constructor(
    idParteneriat: number,
    nomParteneriat: string,
    numeroTelephone: number,
    email: string,
    description: string,
    budget:number,
    verified:boolean
  ) {
    this.idParteneriat = idParteneriat;
    this.nomParteneriat = nomParteneriat;
    this.numeroTelephone = numeroTelephone;
    this.email = email;
    this.description = description;
    this.budget=budget;
    this.verified=verified;
  }
}

File: Backend/partenariat/src/main/java/tn/esprit/partenariat/entities/Parteneriat.java
package tn.esprit.partenariat.entities;

import lombok.*;

import javax.persistence.*;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
@Entity
@Table(name = "parteneriat")
public class Parteneriat {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "idParteneriat", nullable = false)
    private Long idParteneriat;

    @Column(name = "nom_parteneriat")
    private String nomParteneriat;

    @Column(name = "numero_telephone")
    private Long numeroTelephone;

    @Column(name = "email")
    private String email;

    @Column(name = "budget")
    private Long budget;

    @Column(name = "description")
    private String description;

}